testsuite/.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
```

Comment:
I would be very cautious using auto-gen-config.
This can hide important issues just by excluding Cops.